src/go/rpk/pkg/cli/cluster/config/get.go

```diff
@@ -69,3 +89,54 @@ output, use the 'edit' and 'export' commands respectively.`,
 
 	return cmd
 }
+
+func getCloudConfig(ctx context.Context, cfg *config.Config, p *config.RpkProfile, configName string) (string, error) {
+	cloudClient := publicapi.NewCloudClientSet(cfg.DevOverrides().PublicAPIURL, p.CurrentAuth().AuthToken)
+	req := connect.NewRequest(&controlplanev1.GetClusterRequest{
+		Id: p.CloudCluster.ClusterID,
+	})
+
+	cluster, err := cloudClient.Cluster.GetCluster(ctx, req)
+	if err != nil {
+		var ce *connect.Error
+		if errors.As(err, &ce) {
+			if ce.Code() == connect.CodePermissionDenied {
+				return "", fmt.Errorf("this user does not have permission to get cluster information, please check your role or contact admin")
+			}
+			if ce.Code() == connect.CodeNotFound {
+				return "", fmt.Errorf("cluster not found. Please ensure the cluster exists in the Redpanda Cloud")
```

Comment:
```suggestion
				return "", fmt.Errorf("cluster not found. Please ensure the cluster exists in Redpanda Cloud")
```
would it make sense to specify "Redpanda Cloud org"?

Reply:
Yes, it would

Reply:
```suggestion
				return "", fmt.Errorf("cluster not found. Please ensure the cluster exists in your Redpanda Cloud org")
```